docs/07_how-to/13_nested objects.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: 'How to manage nested objects'
+sidebar_label: 'Manage nested objects'
+id: ht-nested-objects
+keywords: [nested objects]
+tags:
+    - nested objects
+
+---
+
+
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+## Nested objects
+Financial instruments often have lengthy and complicated structures. For example, a __Swap Trade__ has multiple __Legs__, each with multiple
+__Cashflows__; and each __Cashflow__ can have multiple __Fixings__.
+
+Nested objects are common methods of incoporating this complexity into financial applications. 
+
+To show you how to create these nested objects, we shall use the example of a __Counterparty Group__, which can contain one or more __Subsidiaries__.  Each __Subsidiary__ can contain one or more __Contacts__.
+
+## Atomic updates
+It is often necessary to modify a nested object as an atomic action, rather than changing the individual objects separately. Genesis supports this capability. 
+
+From applications generated using Genesis Create, you can modify the individual objects. You need to change this to allow an atomic change of the nested object.
+
+To do this, simply declare a Data Access Object (DAO). You can then pass this DAO to events
```

Comment:
DAOs are that created from tables in genesis dictionary. Example is a bit odd as you are declaring them each just in code not via dictionary